app/docker/containers/components/ContainersDatatable/ContainersDatatableActions.tsx

```diff
@@ -36,13 +36,13 @@ export function ContainersDatatableActions({
 }: Props) {
   const selectedItemCount = selectedItems.length;
   const hasPausedItemsSelected = selectedItems.some(
-    (item) => item.Status === 'paused'
+    (item) => item.State === 'paused'
   );
   const hasStoppedItemsSelected = selectedItems.some((item) =>
-    ['stopped', 'created'].includes(item.Status)
+    ['stopped', 'created'].includes(item.State)
```

Comment:
was this a bug until now? or is it just being renamed?

Reply:
a bug